core/src/test/kotlin/com/karumi/kotlinsnapshot/core/InferenceNameTest.kt

```diff
@@ -0,0 +1,39 @@
+package com.karumi.kotlinsnapshot.core
+
+import com.karumi.kotlinsnapshot.core.exceptions.TestNameNotFoundException
+import org.junit.Test
+
+class InferenceNameTest {
+
+    private val camera = Camera()
+
+    @Test
+    fun the_snap_test_name_will_be_inferred_in_test_cases_named_with_test_if_it_is_not_specified(
```

Comment:
In kotlin you can add spaces in the method names, is it this feature working in the library? for example: 

```kotlin
fun `the snap test name will be inferred`
```

Reply:
Thanks! I didn't remember this feature. Let's see if the feature is supported 😃

Reply:
Thank you! I hope to help you soon!